src/bugsnag/ErrorPrompt.tsx

```diff
@@ -0,0 +1,123 @@
+// NOTE: do not include any other files here, try to keep this component separate from the rest
+
+import React from "react";
+
+import "./ErrorPrompt.scss";
+
+const CHUNK_NOT_IN_CACHE = /Loading chunk [\d]+ failed/;
```

Comment:
Can you match on the ChunkError part itself?

Reply:
Chunk error is matched, but it may also be raised if there's some other error, not always related to missing file. This test as described in https://github.com/webpack/webpack.js.org/issues/4479#issuecomment-649416798 helps distinguish if it's a missing file vs invalid contents.

Reply:
Granted, we may deal without it, since the original comment was for a service worker use case , but still, it does provide for more information on the cause of the error.

Reply:
Noted also at https://github.com/sparkletown/sparkle/pull/1858/files#r672249044